### PR TITLE
fix(log): remove reference to sfl4j version

### PR DESCRIPTION
**Pull request by tbouffard**
It is wrong and we do not want to have to maintain it in the documentation.
Also fix some broken link to the Bonita Engine source code.

### CODE REVIEW

File: md/technical-logging.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ The logged messages are not intended to be accessible through the Engine API (se
 ## Usage
 
 If you are creating a custom implementation of a service, you might be interested in using the technical logging
-service. Refer to the implementations of existing services for examples of technical logging service usage (for example the [AuthenticationServiceImpl](https://github.com/bonitasoft/bonita-engine/blob/master/services/bonita-authentication/bonita-authentication-api-impl/src/main/java/org/bonitasoft/engine/authentication/impl/AuthenticationServiceImpl.java)).
+service. Refer to the implementations of existing services for examples of technical logging service usage (for example
+the [AuthenticationServiceImpl](https://github.com/bonitasoft/bonita-engine/blob/7.4.3/services/bonita-authentication/bonita-authentication-api-impl/src/main/java/org/bonitasoft/engine/authentication/impl/AuthenticationServiceImpl.java)).
```
Comment (educhastenier) [inline md/technical-logging.md:15]: :question: why do you want to depend on 7.4.3 ???!!!

File: md/technical-logging.md [hunk 2]
```diff
@@ -20,18 +21,16 @@ Basically, the technical logging service exposes a method that receives the foll
 * throwable (when message is associated with an exception)
 * severity
 
-See the [service
-interface](https://github.com/bonitasoft/bonita-engine/blob/master/services/bonita-log/bonita-log-technical-api/src/main/java/org/bonitasoft/engine/log/technical/TechnicalLoggerService.java) for details.
+See the [service interface](https://github.com/bonitasoft/bonita-engine/blob/7.4.3/services/bonita-log/bonita-log-technical-api/src/main/java/org/bonitasoft/engine/log/technical/TechnicalLoggerService.java) for details.
```
Comment (educhastenier) [inline md/technical-logging.md:24]: same remark?

File: md/logging.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ All the Engine services call the [technical logger service](technical-logging.md
 
 ### Bonita BPM Portal
 
-The Portal uses the [Java Util Logging](http://docs.oracle.com/javase/6/docs/api/java/util/logging/package-summary.html) (JUL) directly to log messages.
+The Portal uses the [Java Util Logging](http://docs.oracle.com/javase/8/docs/api/java/util/logging/package-summary.html) (JUL) directly to log messages.
```
Comment (educhastenier) [inline md/logging.md:15]: nice catch :+1:

File: md/technical-logging.md (hunk 1 above)
Comment (tbouffard) [reply md/technical-logging.md:15]: The link to code on the master branch is broken. To ensure this won't occur in the future, I have to reference code in a tag (which won't change). As the target documentation is 7.4, I choose the latest 7.4.x tag I know.
For 7.5+ doc, we may also point to the right tag or keep the 7.4.x

**Review by educhastenier — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**